airbrake.gemspec

```diff
@@ -31,7 +31,7 @@ DESC
 
   s.required_ruby_version = '>= 2.0'
 
-  s.add_dependency 'airbrake-ruby', '~> 1.6'
+  # s.add_dependency 'airbrake-ruby', '~> 1.6'
```

Comment:
This is temporary, so I can depend on unreleased features (will be merged after airbrake-ruby is released).